docs/tutorials/projects/integrating-viam-with-openai.md

```diff
@@ -266,31 +250,30 @@ Click **Save config** to finish adding the detector.
 With the rover and tutorial code set up and it is time to bring your companion robot to life!
 Let's call her "Rosey", and bring her to life by running:
 
-```bash
+``` bash
```

Comment:
For code review, need help: I think my linter did this, I'm not sure if good or bad/should remove or not?

Reply:
Same with everything down to `Alternative Option` header